addon/models/user.js

```diff
@@ -1,8 +1,8 @@
 import DS from 'ember-data';
 
-import OsfModel from 'ember-osf/models/base';
+import OsfModel from '../mixins/osf-model';
```

Comment:
I think they're equivalent ways of importing the file, but for whatever reason '../' in an import statement has always made me a bit anxious. That said many of the ember blueprints seem to do this...

No change needed, but we should try to be consistent with this going forward.